tests/playwright/src/model/pages/volumes-page.ts

```diff
@@ -58,6 +59,20 @@ export class VolumesPage extends MainPage {
     return new VolumeDetailsPage(this.page, volumeName);
   }
 
+  async deleteVolume(volumeName: string): Promise<VolumesPage> {
+    const volumeRow = await this.getVolumeRowByName(volumeName);
+    if (volumeRow === undefined) {
+      throw Error(`Volume: ${volumeName} does not exist`);
+    }
+    const containerRowDeleteButton = volumeRow.getByRole('button', { name: 'Delete Volume' });
+    await playExpect(containerRowDeleteButton).toBeVisible();
+    await playExpect(containerRowDeleteButton).toBeEnabled();
```

Comment:
There isn't a point in using toBeVisible() before toBeEnabled(), since toBeEnabled() checks for visibility implicitly.